lib/web/integrations_awsoidc.go

```diff
@@ -1041,6 +1041,18 @@ func (h *Handler) awsOIDCCreateAWSAppAccess(w http.ResponseWriter, r *http.Reque
 		return nil, trace.Wrap(err)
 	}
 
+	// If the integration name contains a dot, then the proxy must provide a certificate allowing *.<something>.<proxyPublicAddr>
+	if strings.Contains(integrationName, ".") {
```

Comment:
This check does not conform with the check that was added in `CreateIntegration()`. Does that mean it is now possible to have an existing integration name that produces an invalid dns name which will bypass this check? Should this be doing the same validation as `CreateIntegration()`?

Reply:
> Does that mean it is now possible to have an existing integration name that produces an invalid dns name which will bypass this check?

I'm trying to be less intrusive on existing integrations, to prevent breaking existing integrations.
So, while the checks are not the same, this will still be a valid DNS name because creating an AppServer will ensure the DNS name is valid.